src/mentors.json

```diff
@@ -1,12 +1,17 @@
-[{
+[
+  {
     "id": "moshfeu@gmail.com",
     "name": "Mosh Feuchtwanger",
     "avatar": "https://avatars3.githubusercontent.com/u/3723951?s=460&v=4",
     "title": "Frontend at @wix",
     "description": "Hi, I'm Mosh and I will love to help! 🤪",
     "country": "Israel",
     "tags": [
-      "frontend", "javascript", "reactjs", "html", "css"
```

Comment:
Maybe soon we will adopt a prettier rule for this file. In a minewhile, let's keep the format the way it is. Thanks!